_posts/2020-09-09-codebuild-reports-cypress-e2e-tests.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: "Adding CodeBuild reports to Cypress e2e tests using CDK"
+date: 2020-09-09
+tags: aws codebuild reports testing devops
+author: Natalie Laing
+image: /img/reports.png
+---
+
+![Generic reports](/img/reports.png)
+
+## Introduction
+
+I was recently working on a project where the project owner wanted to be able to visualize our tests in one place without having to navigate through the codebase.
+
+This enabled us to visualize the following in our report group:
+
+- Pass rate
+- Average report duration
+- Average number of test cases run
+- Number of reports generated
+
+CodeBuild reports allow you to drill deeper into each report generated by visualizing:
```

Comment:
Why do these two sections appear so similar? Could they be combined or presented differently?